examples/discovery-search-app/cypress/integration/passages/spec.ts

```diff
@@ -99,5 +99,50 @@ describe('Passage Results', () => {
       cy.get('@combinedResult').contains('View passage in document');
       cy.get('@combinedResult').contains('View table in document');
     });
+
+    describe('and clicking on "CI Document" for a result', () => {
+      beforeEach(() => {
+        cy.get('button[data-testid="search-result-element-preview-button"]')
+          .contains('View table in document')
+          .click();
+        cy.get('ul')
+          .contains('CI')
+          .should('exist');
+        cy.get('ul')
+          .contains('CI')
+          .click();
+      });
+
+      it('verify that CI Document sections are present', () => {
+        cy.get('.bx--document-preview').should('exist');
+        cy.get('.bx--ci-doc__sidebar').should('exist');
+        cy.get('.bx--ci-doc__doc').should('exist');
+        cy.get('.bx--ci-doc__details').should('exist');
```

Comment:
Again here, this seems fragile (although it looks like others are doing it as well). I really like the `react-testing-library` ethos, wherein the test should try to only use user-visible constructs (text, buttons, links) or `data-testid` as a last resort.

I believe we've done similar checks in our unit tests where we just looked for the appropriate (unique) text. We should do the same here and below.